### ISSUE TITLE
Adding Damage Per Minute to Match Overview and Records

### ISSUE THREAD

**Issue by elkcarc** [odota/web#1834]
In most other competitive games DPM or ADR (Average Damage per Round) is an important metric by which player performance is measured. Although the current system is adequate, it would be nice to see damage stats independent of game length.

**Comment by howardchung** [comment]
we could probably compute DPM on the client side and display it somewhere

**Comment by PowerSlime** [comment]
There is "Hero Damage per Minute" stats in "Benchmarks" section
![image](https://user-images.githubusercontent.com/18900377/54497934-49035800-4909-11e9-95e2-633f2d2c0691.png)
Or do you mean about viewing DPM at the specific time interval?

**Comment by howardchung** [comment]
That's true, I forgot we compute it in that section. I'm guessing this was what was asked for.

**Comment by elkcarc** [comment]
Since it is already computed, could it be added to the records section so you could see highest all time dpm stats and filter to see highest dpm by hero?

**Comment by howardchung** [comment]
The problem is it won't be backfilled in Cassandra for old matches so the records would only be from now on
